modules/nf-core/gatk4/printsvevidence/main.nf

```diff
@@ -8,17 +8,16 @@ process GATK4_PRINTSVEVIDENCE {
         'quay.io/biocontainers/gatk4:4.2.6.1--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(evidence), path(evidence_index)
+    tuple val(meta), path(evidence_files), path(evidence_indices)
     path bed
     path fasta
     path fasta_fai
     path dict
 
-
     output:
-    tuple val(meta), path("*.txt.gz")       , emit: printed_evidence
-    tuple val(meta), path("*.txt.gz.tbi")   , emit: printed_evidence_index
-    path "versions.yml"                     , emit: versions
+    path "*.txt.gz"       , emit: printed_evidence
+    path "*.txt.gz.tbi"   , emit: printed_evidence_index
```

Comment:
I think we should keep the meta map in output as well

Reply:
Oh yeah, forgot to re-add them there. my bad :p